metis-normalization/src/main/java/eu/europeana/normalization/dates/extraction/extractors/LongNegativeYearDateExtractor.java

```diff
@@ -0,0 +1,36 @@
+package eu.europeana.normalization.dates.extraction.extractors;
+
+import eu.europeana.normalization.dates.DateNormalizationExtractorMatchId;
+import eu.europeana.normalization.dates.DateNormalizationResult;
+import eu.europeana.normalization.dates.edtf.InstantEdtfDate;
+import eu.europeana.normalization.dates.edtf.InstantEdtfDateBuilder;
+import eu.europeana.normalization.dates.extraction.DateExtractionException;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * A year before 1 AD with more than 4 digits. This pattern is typically used in archaeological contexts. The year may contain
+ * between 5 and 9 digits. Aso includes the pattern for ranges of this kind of years.
+ */
+public class LongNegativeYearDateExtractor extends AbstractDateExtractor {
+
+  private static final Pattern YEAR_PATTERN = Pattern.compile("(-?\\d{5,9})");
+
+  @Override
+  public DateNormalizationResult extract(String inputValue,
+      boolean flexibleDateBuild) throws DateExtractionException {
```

Comment:
suggestion also to update the variable `flexibleDateBuild`for `allowDayMonthSwap`

Reply:
done